widgets/src/_standalone/ProcessOverview/index.svelte

```diff
@@ -58,8 +80,17 @@
 				>{total ?? '?'}</span
 			>
 		</div>
-		<div class="p-4">
+		<div class="p-4 min-h-[450px] flex flex-col justify-between">
 			<Table columns={data.columns} rows={data.rows}></Table>
+			{#if total !== null}
+				<Pagination
+					totalAmountOfButtons={Math.ceil(total / size)}
+					totalAmount={total}
```

Comment:
Is there any specific reason for not just calling it `total`?

```suggestion
					{total}
```